server/src/dtos/plugin.dto.ts

```diff
@@ -1,9 +1,16 @@
 import { IsNotEmpty, IsString } from 'class-validator';
 import { PluginAction, PluginFilter } from 'src/database';
-import { PluginContext } from 'src/enum';
+import { PluginContext as PluginContextType, PluginTriggerType } from 'src/enum';
 import type { JSONSchema } from 'src/types/plugin-schema.types';
 import { ValidateEnum } from 'src/validation';
 
+export class PluginTriggerResponseDto {
```

Comment:
Same here. IMO they are triggers for workflows (`WorkflowTrigger`), they just happen to be provided by plugins.